spec/migrations/20200325170358_add_domain_and_relative_path_to_automate_spec.rb

```diff
@@ -0,0 +1,75 @@
+require_migration
+
+describe AddDomainAndRelativePathToAutomate do
+  let(:dom_stub) { migration_stub(:MiqAeDomain) }
+  let(:ns_stub) { migration_stub(:MiqAeNamespace) }
+  let(:class_stub) { migration_stub(:MiqAeClass) }
+  let(:method_stub) { migration_stub(:MiqAeMethod) }
+  let(:instance_stub) { migration_stub(:MiqAeInstance) }
+
+  migration_context :up do
+    # nodes:
+    # dom1
+    #   ns11
+    #     ns111
+    #     ns112
+    # dom2
+    #   ns21
+    #     class211
+    #       instance2111
+    #       method2112
+    it "updates tree" do
```

Comment:
Can we get some more specs with degenerate cases (0 children) and cases with more than 1 child?  Things like

- domains without namespaces
- domains without multiple namespaces
- namespaces without classes
- namespaces with multiple classes
- etc.